ps_facetedsearch-attribute-indexer.php

```diff
@@ -17,6 +17,14 @@
  * @copyright Since 2007 PrestaShop SA and Contributors
  * @license   https://opensource.org/licenses/AFL-3.0 Academic Free License 3.0 (AFL-3.0)
  */
+
+/*
+ * This standalone endpoint is deprecated, it should not be sued anymore and should be removed along with the
```

Comment:
```suggestion
 * This standalone endpoint is deprecated, it should not be used anymore and should be removed along with the
```
Yes please don't sue it the poor guy